app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Kindly add the `foreign_key` to each relation, to `author_id`. So in this way, the many relations will be referenced to author_id

Reply:
Alright, Thanks for the correction.